### ISSUE TITLE
.day() mistake over fall dst change

### ISSUE THREAD

**Issue by craigpaul** [spencermountain/spacetime#236]
I encountered an odd problem this morning (which I believe to be a very small edge case potentially). When I am using a custom start of the week (Monday for ISO purposes) and ask for the start of the week with the date being November 1st, 2020, it (which is actually a Sunday of course), tells me that is the start of the week.

```js
spacetime('2020-11-01').weekStart(1).startOf('week').format('iso-short') // '2020-11-01'
```

That _should_ instead be returning Oct 26th, 2020 I believe. Any thoughts? Potentially to do with the DST shift? I should mention, I'm in the `America/Regina` timezone, and I don't experience this, but a work colleague in the `America/Vancouver` timezone does experience this. If I change my OS timezone to `America/Vancouver` I do see this happen.

**Comment by spencermountain** [comment]
whoa, yeah. this is bad.

```js
spacetime('2020-11-01').day('monday').format('{day} {nice}') // 'sunday nov 1st'
```

**Comment by spencermountain** [comment]
got a fix on dev, will be part of the next release, hopefully this week.

**Comment by craigpaul** [comment]
Awesome, thanks for the quick fix Spencer!

**Comment by spencermountain** [comment]
fixed in `6.9.0`
